pkg/multicluster/kubeconfig/kubeconfig_secret_test.go

```diff
@@ -32,7 +32,6 @@ preferences: {}
 users:
 - name: test-name
   user:
-    as-user-extra: null
```

Comment:
why were a bunch of files under `pkg/multicluster` deleted? are we sure we don't need those anymore?
and why was this line removed?

Reply:
That is a great question that I don't immediately have the answer to. Looks like those files were deleted as part of Eitan's initial commits. @EItanya do you have context on why those files were removed?